engine/security-api/src/main/scala/pl/touk/nussknacker/ui/security/api/PermissionSyntax.scala

```diff
@@ -5,17 +5,16 @@ object PermissionSyntax {
   implicit class UserPermissionsQuery(user: LoggedUser) {
     def can(permission: Permission.Permission): Set[String] =
```

Comment:
If we make some cleaning maybe let's remove/change this method? It's a bit misleading in admin context - in fact for admin we want *all* categories and not only those to which admin has explicit permissions? See usages

Reply:
I agree.. But if we want do this good, than we should create separated config which has all available categories.. It shoud be more complex solutions - now I want only fix it ;)